aperturedb/BBoxLoader.py

```diff
@@ -140,6 +140,7 @@ def generate_batch(self, bbox_data):
             fi = {
                 "FindImage": {
                     "_ref": img_ref,
+                    "blobs": False,
```

Comment:
this is another unrelated bug fix, where boundingbox loader is unnecessarily retrieving img blobs.